health/REFERENCE.md

```diff
@@ -129,6 +132,32 @@ You're interested in what comes after the comma: `disk.io`. That's the name of t
 
 If you create a template using the `disk.io` context, it will apply an alarm to every disk available on your system.
 
+#### Alarm line `class`
+
+Specify the classification of the alarm or template. 
+
+Class can be used to indicate the broader area of the system that the alarm applies to. For example, under the general `Database` class, you can group together alarms that operate on various database systems, like `MySQL`, `CockroachDB`, `CouchDB` etc.
+
+```yaml
+class: Database
```

Comment:
What about allowed characters? Are there any restrictions? Are spaces allowed?

Reply:
Considering the current code in https://github.com/netdata/netdata/pull/10961, I think we would have the same restrictions that JSON has, except for quotes that we are stripping, @MrZammler can you confirm this?

Reply:
Hi Thiago, yes, I will do some tests and report back.

Reply:
I will wait for your tests before to proceed with the review, the question @ilyam8 raised is very important for that PR.
Please, also take a look in the next question I wrote, because it also impacts the PR.

Reply:
The final values of these strings are consumed by the cloud. I think they're used to group together alarms. That's why I've added @dim08 (the docs should reflect that too - that's their main use after all). 

So I think if the user chooses to have a `class` value of `System 1 - my great machine!!^@&#` to some alarms, and the cloud is able to consume it, we should allow it to be read and pass through the agent. We do strip quotes (both single and double), so this could go in the docs maybe? (But again if grouping is the target, then it should work).

The same could be true for the info field? I think any character that can be passed through JSON should be allowed, unless the cloud has other restrictions that I'm not aware of. Of course some kind of non-utf valid character, will likely break things, so if it's needed we should sanitize the fields (and the info one too).

Reply:
> What about allowed characters? Are there any restrictions? Are spaces allowed?

I do not know exactly what the product team wants, but I made some tests on Friday and today, and I saw that we can use any character for these events, on the other hand I discovered a bug thanks these tests, but I will bring a complete different PR that will address it.

Please, talk with the cloud team to confirm they are expecting any ASCII character.

Reply:
@MrZammler the unique doubt I have to approve the PR is this discussion, do we arrive in the conclusion that cloud is allowing any ASCII character?

Reply:
Hi Thiago, I'm looking into this (the characters), thanks.

Reply:
Sorry Thiago for being late to address this.... 

So, I did a test on the info field (I can't find the class/component/type strings on the cloud...), and the cloud will display an info string with e.g. `number !@#$%^&*()_+=-}{][;:?/><,. of metrics lost ....`. I've asked on cloud-ops, will update.

Reply:
Furthermore, I've received an alert email from the cloud, after having set a class attribute to ` Classification: This is a class !@#$%^*()_&+=-}{][:;?><,./ ` and it correctly appeared in the email message: